diff --git a/mllm/mllm.cpp b/mllm/mllm.cpp
--- a/mllm/mllm.cpp
+++ b/mllm/mllm.cpp
@@ -59,7 +59,7 @@ void shutdownContext() {
 
   // Clean up memory before backend is freed.
   // FIXME:
   // This line is needed for cuda !!!
-  // Context::instance().memoryManager()->clearAll();
+  Context::instance().memoryManager()->clearAll();
 }
 
 void setLogLevel(const LogLevel& level) { ::mllm::Logger::level() = level; }
